.github/scripts/publish-rhel.sh

```diff
@@ -57,38 +56,49 @@ wait_for_container_scan()
         return 0
     fi
 
-    local NOF_RETRIES=$(( TIMEOUT_IN_MINS / 2 ))
-    # Wait until the image is scanned
-    for i in $(seq 1 "${NOF_RETRIES}"); do
+    # start timer
+    _start_stopwatch
+
+    while [ 1 ]
+    do
         local IMAGE
         local SCAN_STATUS
         local IMAGE_CERTIFIED
+        local RESULT=-1
 
         IMAGE=$(get_image not_published "${RHEL_PROJECT_ID}" "${VERSION}" "${RHEL_API_KEY}")
         SCAN_STATUS=$(echo "${IMAGE}" | jq -r '.data[0].container_grades.status')
         IMAGE_CERTIFIED=$(echo "${IMAGE}" | jq -r '.data[0].certified')
 
-        if [[ ${SCAN_STATUS} == "pending" ]]; then
+        if _is_stopwatch_expired; then
+            RESULT=42
+            echo "Timeout'! Scan could not be finished"
```

Comment:
```suggestion
            echo "Timeout! Scan could not be finished"
```

Also might want to consider using `echoerr` instead.

Reply:
Good catch

Fixed https://github.com/hazelcast/hazelcast-docker/pull/868/commits/4ad35d0502d6b49d3b8136ae1056cce8866133e8 and https://github.com/hazelcast/hazelcast-docker/pull/868/commits/da2de66e74f7087d40a3df1e92d7be877cfc0b16